sentry-android-integration-tests/sentry-uitest-android/src/androidTest/java/io/sentry/uitest/android/EnvelopeTests.kt

```diff
@@ -56,4 +62,31 @@ class EnvelopeTests : BaseUiTest() {
             assertNoOtherRequests()
         }
     }
+
+    @Test
+    fun sendProfiledTransaction() {
+        // This is a dogfooding test
+        IdlingRegistry.getInstance().register(ProfilingSampleActivity.scrollingIdlingResource)
+        initSentry(false) { options: SentryOptions ->
+            options.dsn = "https://640fae2f19ac4ba78ad740175f50195f@o1137848.ingest.sentry.io/6191083"
+            options.tracesSampleRate = 1.0
+            options.isProfilingEnabled = true
+        }
+
+        val transaction = Sentry.startTransaction("e2etests", "testProfile")
+        val benchmarkScenario = launchActivity<ProfilingSampleActivity>()
+        swipeList(10)
+        benchmarkScenario.moveToState(Lifecycle.State.DESTROYED)
+        transaction.finish()
+        Thread.sleep(100)
```

Comment:
`l`: Why do we need these `sleep`? Can you add the info as a comment, please?

Reply:
useless, actually. i removed it